web/client/observables/__tests__/wfs-test.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020, GeoSolutions Sas.
```

Comment:
new files should contain current year, please apply this to all new files
```suggestion
 * Copyright 2024, GeoSolutions Sas.
```

Reply:
Included in https://github.com/geosolutions-it/MapStore2/pull/10004/commits/22f0d9ba262e3352d0572dcd6c748162626449d7